### PR TITLE
TIP-831: Upgrade to jQuery 3.3.1

### PR DESCRIPTION
**Pull request by tamarasaurus**
**Description (for Contributor and Core Developer)**

- [x] jquery 1.11.1 -> 1.12.4
- [x] jquery 1.12.4 - > 2.2.4
- [x] jquery  2.2.4 -> 3.0.0
- [x] jquery-ui 1.11.4 -> 1.12.1
- [x] check plugins
- [x] check bootstrap
- [x] check EE

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product-model/form/add-child/fields-container.js [hunk 1]
```diff
@@ -123,12 +126,24 @@ define(
...
+             * Render validation errors for fields after field extensions are loaded
+             * @return {[type]} [description]
+             */
+            showErrors() {
+                this.getRoot().trigger('pim_enrich:form:entity:bad_request',
+                    { sentData: this.getFormData(), response: this.errors }
+                );
+            },
+
+
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product-model/form/add-child/fields-container.js]: Double line

File: src/Pim/Bundle/UIBundle/Resources/public/lib/jquery/jquery-1.10.2.js [hunk 2]
```diff
@@ -1,9789 +0,0 @@
...
-			return rheader.test( elem.nodeName );
-		},
-
-		"input": function( elem ) {
-			return rinputs.test( elem.nodeName );
-		},
-
-		"button": function( elem ) {
-			var name = elem.nodeName.toLowerCase();
-			return name === "input" && elem.type === "butt
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/lib/jquery/jquery-1.10.2.js]: OK I read every line of this file and it's ok

File: src/Pim/Bundle/UIBundle/Resources/public/lib/backbone.bootstrap-modal.js [hunk 3]
```diff
@@ -1,284 +0,0 @@
...
-    <% if (title) { %>\
-      <div class="modal-header">\
-        <% if (allowCancel) { %>\
-          <a class="close">×</a>\
-        <% } %>\
-        <h3>{{title}}</h3>\
-      </div>\
-    <% } %>\
-    <div class="modal-body">{{content}}</div>\
-    <div class="AknButtonList AknButtonList--right modal-footer">\
```
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/lib/backbone.bootstrap-modal.js]: Is it still used ?

Comment (tamarasaurus) [reply src/Pim/Bundle/UIBundle/Resources/public/lib/backbone.bootstrap-modal.js]: So the other bootstrap-modal.js is more recent and includes an if statement around this block - https://github.com/akeneo/pim-community-dev/blob/2.1/src/Pim/Bundle/UIBundle/Resources/public/lib/bootstrap-modal.js#L35 It seems ok to use, didn't see any problem ¯\_(ツ)_/¯

Comment (pierallard) [reply src/Pim/Bundle/UIBundle/Resources/public/lib/backbone.bootstrap-modal.js]: ¯\(ツ)/¯

**Comment by tamarasaurus** [comment]
To be fixed for 2.3

**Pull request closed without merging**